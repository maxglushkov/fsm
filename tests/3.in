MEALY
9
0
2
