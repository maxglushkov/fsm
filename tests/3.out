Mealy
1
0
2
